src/test/java/com/google/vinet/servlets/LogoutServletTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.vinet.servlets;
+
+import com.google.appengine.api.users.UserService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.anyString;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class LogoutServletTest {
+  @Mock UserService userService;
+
+  @Mock HttpServletRequest request;
+
+  @Mock HttpServletResponse response;
+
+  /* The @InjectMocks annotation instructs Mockito to inject the mocked
```

Comment:
Is this not obvious?

Reply:
Potentially, though I don't think including it will do any harm for someone less familiar with the Mockito framework.

I can remove the comment though if we feel its inclusion harms the readability of the code, rather than aiding it.

Do you think it serves to negatively impact readability?

Reply:
I don't think it does any more than any other redundant comment, no.